package-lock.json

```diff
@@ -7917,11 +7953,13 @@
             },
             "wrappy": {
               "version": "1.0.2",
-              "bundled": true
+              "bundled": true,
+              "optional": true
             },
             "yallist": {
               "version": "3.0.3",
-              "bundled": true
+              "bundled": true,
+              "optional": true
```

Comment:
Quick tip: it looks like some of you folks have different `npm` versions. You can use an `engines` property in your `package.json` to help stop these weird unrelated changes: https://docs.npmjs.com/files/package.json#engines

Reply:
Interesting, we've been wondering why this kept happening. Not sure what node version everyone is using at the moment. Would allowing  > 8 be ok for now?